app/importers/merchants_importer.rb

```diff
@@ -0,0 +1,36 @@
+require 'csv'
+
+class MerchantsImporter
+  attr_reader :filename,
+              :number_of_lines
+
+  def initialize(filename)
+    @filename = filename
+    @number_of_lines = `wc -l #{filename}`.to_i
+  end
+
+  def import
+    bar = ProgressBar.create(title: "Merchants", total: number_of_lines)
+
+    CSV.foreach(filename, headers: true) do |row|
+
+      merchant_attributes = {
+        id:         row['id'].to_i,
+        name:       row['name'],
+        updated_at: to_date(row['updated_at']),
+        created_at: to_date(row['created_at'])
+      }
+            
+      Merchant.create!(merchant_attributes)
```

Comment:
@ski-climb Ok sounds good. Yea I agree on the test. I'm going to work on that now.